src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,70 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
+            where TNode : SyntaxNode
+        {
+            bool result = true;
+            for (int i = 0; i < items.Count; i++)
+            {
+                result &= predicate(items[i]);
+            }
+
+            return result;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxToken token)
+            => token.Parent is AccessorListSyntax accessorList &&
+                All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+            => node?.Parent is AccessorListSyntax accessorList &&
+                All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool HasInitializer(BasePropertyDeclarationSyntax basePropertyDeclaration)
+            => (basePropertyDeclaration is PropertyDeclarationSyntax property && property.Initializer != null);
```

Comment:
```suggestion
            => basePropertyDeclaration is PropertyDeclarationSyntax property && property.Initializer != null;
```

Reply:
I have difficulties in remembering the formatting rules with every customer I work with :)
Each of them use different styles.
Do you want me to merge this one? This will re-trigger everything

Reply:
> Do you want me to merge this one?

Yes please :)

> This will re-trigger everything

C'est la vie :)